dask_planner/src/sql.rs

```diff
@@ -140,12 +140,21 @@ impl ContextProvider for DaskSQLContext {
         }
 
         // Loop through all of the user defined functions
-        for schema in self.schemas.values() {
+        let schemas = self.schemas.clone();
+        for schema in schemas.values() {
```

Comment:
We no longer need this clone now that the function hash map contains `Arc` references that we clone before passing to the closure.

```suggestion
        for schema in self.schemas.values() {
```